server/lib/tuist/authorization.ex

```diff
@@ -287,14 +287,6 @@ defmodule Tuist.Authorization do
       allow([:authenticated_as_user, user_role: :admin])
     end
 
-    action :update_repository do
```

Comment:
you can no longer update a repository this way, so we don't need the repository permission check anymore